ouroboros-consensus/src/Ouroboros/Consensus/NodeNetwork.hs

```diff
@@ -392,9 +404,9 @@ consensusNetworkApps
        )
     => NodeKernel m peer blk
     -> ProtocolTracers m peer blk failure
-    -> ProtocolCodecs blk failure m bytesCS bytesBF bytesTX bytesLCS bytesLTX
+    -> ProtocolCodecs blk failure m bytesCS bytesBF bytesBF bytesTX bytesLCS bytesLTX
     -> ProtocolHandlers m peer blk
-    -> NetworkApplication m peer bytesCS bytesBF bytesTX bytesLCS bytesLTX ()
+    -> NetworkApplication m peer bytesCS bytesBF bytesBF bytesTX bytesLCS bytesLTX ()
```

Comment:
By using the same `bytesBF` parameter for both the bytes of `BlockFetch` and `BlockFetchSerialised` I require them to have the same type for serialised messages.